lib/modules/ai_bot/commands/read_command.rb

```diff
@@ -50,7 +50,15 @@ def process(topic_id:, post_number: nil)
 
       posts = posts.where("post_number = ?", post_number) if post_number
 
-      content = +"title: #{topic.title}\n\n"
+      content = +<<~TEXT.strip
+        title: #{topic.title}
+        category: #{topic.category&.name}
```

Comment:
Can we include the parent category too?

Reply:
interesting, yes we can and probably should! will change.